flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/WindowedStream.java

```diff
@@ -124,16 +124,17 @@ public WindowedStream(KeyedStream<T, K> input,
 	}
 
 	/**
-	 * Sets the allowed lateness. If the {@link WindowAssigner} used
+	 * Sets the allowed lateness. By default this is 0. If the {@link WindowAssigner} used
```

Comment:
The javadoc is out of date with the new exception-throwing behavior. For the text of the exception I would prefer something like "Setting the allowed lateness is only valid for event-time windows.". Precise and simple. 😃